templates/pages/contact.html

```diff
@@ -25,56 +25,114 @@
-            {% else %}
+            </ul>
+        {% else %}
+            <p>
                 {% trans "Il n'y a pas de membre dans ce groupe." %}
-            {% endif %}
-        </ul>
+            </p>
+        {% endif %}
+    </div>
+
+    <h3>{% trans "L'équipe de communication" %}</h3>
+    <p>
+        {% blocktrans %}
+            L'équipe de communication gère la communication du site sur les réseaux sociaux et sur la page d'accueil (notamment la partie "A la Une"). Elle a pour but de mettre en avant le contenu et les créations des membres.
+        {% endblocktrans %}
+    </p>
+    <p>
+        {% blocktrans with email_contact=app.site.email_contact|obfuscate_mailto_top_subject:"Contact communication" %}
+            Vous pouvez à tout moment joindre cette équipe par courriel via {{ email_contact }} ou <a href="{{ communication_team_contact_url }}">en leur envoyant un message privé</a>.
+        {% endblocktrans %}
+    </p>
+    <div class="authors">
+        {% if communication_team %}
+            <ul>
+                {% for member in communication_team %}
+                    <li>{% include "misc/member_item.part.html" with avatar=True %}</li>
+                {% endfor %}
+            </ul>
+        {% else %}
+            <p>
+                {% trans "Il n'y a pas de membre dans ce groupe." %}
+            </p>
+        {% endif %}
     </div>
 
     <h3>{% trans "L'équipe technique" %}</h3>
     <p>
         {% blocktrans with repository=app.site.repository %}
             L'équipe technique est constituée de certains membres du site dont le but est d'une part de s'assurer que le site reste toujours disponible en ligne, et d'autre part de corriger les bogues rencontrés sur le site ainsi que d'ajouter de nouvelles fonctionnalités.
-            Des administrateurs systèmes, jusqu'aux designeurs, en passant par les développeurs back-end et intégrateurs front-end, ils s'occupent aussi de la maintenance du <a href="{{ repository }}">dépôt officiel du projet</a>. Les membres faisant partie de l'équipe technique sont les suivants :
+            Des administrateurs systèmes, jusqu'aux designeurs, en passant par les développeurs back-end et intégrateurs front-end, ils s'occupent aussi de la maintenance du <a href="{{ repository }}">dépôt officiel du projet</a>. Pour rester cohérent et organisé, le projet est orchestré par un Directeur Technique Canidé [DTC] et un Chef de Projet [CdP] (chaque développeur reste néanmoins autonome).
+        {% endblocktrans %}
+    </p>
+    <p>
+        {% blocktrans with url_forum=app.site.forum_feedback_users %}
+            Vous pouvez à tout moment joindre cette équipe <a href="{{ devs_contact_url }}">en leur envoyant un message privé</a> sauf pour les bugs et suggestions, auquel cas il est préférable de créer un sujet dans <a href="{{ url_forum }}">ce forum</a>.
         {% endblocktrans %}
     </p>
     <div class="authors">
-        <ul>
-            {% if devs %}
+        {% if devs %}
```

Comment:
Tu peut utiliser la forme raccourci `{% for %} {% empty %}` ([voir doc](https://docs.djangoproject.com/en/1.7/ref/templates/builtins/#for-empty)) ici pour faire moins verbeux.